tests/functional_tests/test_training.py

```diff
@@ -0,0 +1,43 @@
+import os
+import pickle
+import argparse
+import numpy as np
+import pandas as pd
+from scipy.stats import randint
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.impute import SimpleImputer
+from sklearn.linear_model import LinearRegression
+from sklearn.model_selection import GridSearchCV, RandomizedSearchCV
+from sklearn.tree import DecisionTreeRegressor
+
+
+PATH = "/home/deepak/practice_git_repo/assignment2/mle-training/data/processed"
```

Comment:
for pytest cases you should now read from paths. all data has to be pasted as fixture.